yt_dlp/extractor/vidio.py

```diff
@@ -56,9 +60,28 @@ def _real_extract(self, url):
             })
         video = data['videos'][0]
         title = video['title'].strip()
+        hls_url = data['clips'][0]['hls_url']
+
+        formats, subs = self._extract_m3u8_formats_and_subtitles(
+            hls_url, display_id, 'mp4', 'm3u8_native', errnote=False, fatal=False)
+        if not formats:
+            self.to_screen('Falling back to premier mode.')
+            sources = self._download_json(
+                'https://www.vidio.com/interactions_stream.json?video_id=' + video_id + '&type=videos', display_id)
+            if not (sources.get('source') or sources.get('source_dash')):
+                self.raise_login_required()
```

Comment:
```suggestion
                self.raise_login_required(method='cookies')
```

I had added an optional argument `method` to show a more usefull error message to user based on whether cookies or user-pass is the accepted auth method

Reply:
done